code/modules/spells/spell_types/ethereal_jaunt.dm

```diff
@@ -30,6 +30,7 @@
 			animation.layer = 5
 			animation.master = holder
 			target.ExtinguishMob()
+			mutations = list(MUT_MUTE)

```

Comment:
Put a line above this that says
var/mob/living/carbon/wiz = user
Then change "mutations" on this line to "wiz.mutations"

Also this might override any existing defects like monkeyism or hulk/lasereyes from mutate